### ISSUE TITLE
Suggestion: Show recently added resources?

### ISSUE THREAD

**Issue by ravachol** [storypalette/storypalette-editor#30]
After creating a resource, the user has to search for it before being able to add it to the palette.

Couldn't we show "My recently added resources" at the top of the list so that the user can just pick the first one and add it to the palette? If the user is adding many resources at the same time, couldn't we just show a list of these new ones?

**Comment by snawbel** [comment]
Another way could be to automatically add a new resource to the current palette. Optionally with a checkbox "Add to palette". What do you think @sveta-suvorina?

**Comment by sveta-suvorina** [comment]
good idea! seems like in the first option the workflow is to first load all the resources, and then add the right ones in the correct order to the palette. and in the second - first add, then sort if needed. both are fine, it depends on the workflow of the librarians. do we know anything about that? or maybe we can do both - with the checkbox "add to palette", so the resource automatically added to the palette, and also stays on top of the list?

**Comment by ravachol** [comment]
Having both solutions sounds good to me.
